yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -214,3 +227,34 @@ func (adb *AssessmentDB) PopulateMigrationAssessmentStats() error {
 
 	return nil
 }
+
+func (adb *AssessmentDB) FetchAllStats() (*[]MigrationAssessmentStats, error) {
```

Comment:
Let's use the terminology of "table stats" instead of "migration assessment stats" for this piece of information and associated methods.

Reply:
how about TABLE_INDEX_STATS? Since it has combined info for both - indexes and tables
I went with `MigrationAssessmentStats` because i was thinking to maintain this single table moving forward for storing everything related to migration stats. 
Perf lib etc don't need to care about other tables like TABLE_INDEX_IOPS, TABLE_INDEX_SIZE after this table now.